controllers/feeds/getFeedsV2.js

```diff
@@ -237,12 +237,14 @@ module.exports = async (req, res) => {
               postActors.push(item.actor.id);
             }
 
+            const actor_from_reaction = getActorFromLatestReaction(item.latest_reactions);
+            postActors = [...postActors, ...actor_from_reaction];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getActorFromLatestReaction` function is used to get actors from the latest reactions and add them to the `postActors` array. However, it's not clear if this function ensures that the returned actors are unique and not already present in the `postActors` array. If there are duplicates, this could lead to unnecessary computation when calculating karma scores. Consider ensuring uniqueness of actors before adding them to the `postActors` array.
```diff
- const actor_from_reaction = getActorFromLatestReaction(item.latest_reactions);
- postActors = [...postActors, ...actor_from_reaction];
+ const actor_from_reaction = getActorFromLatestReaction(item.latest_reactions);
+ const uniqueActors = [...new Set([...postActors, ...actor_from_reaction])];
+ postActors = uniqueActors;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->